kroxylicious/src/main/java/io/kroxylicious/proxy/future/BrutalFuture.java

```diff
@@ -0,0 +1,31 @@
+package io.kroxylicious.proxy.future;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class BrutalFuture<T> extends CompletableFuture<T> {
+
+    public boolean internalComplete(T value) {
+        return super.complete(value);
+    }
+
+    public boolean internalCompleteExceptionally(Throwable ex) {
+        return super.completeExceptionally(ex);
+    }
+
+    @Override
+    public boolean complete(T value) {
+        throw new RuntimeException("don't complete me");
+    }
+
+    @Override
+    public boolean completeExceptionally(Throwable ex) {
+        throw new RuntimeException("don't complete me");
+    }
+
+    @Override
+    public T get() throws InterruptedException, ExecutionException {
```

Comment:
should we not support get(long timeout, TimeUnit unit) and join too?

Reply:
added `get` and `join`